core/src/commonMain/kotlin/com/ustadmobile/core/contentformats/xapi/endpoints/XapiStatementEndpointExt.kt

```diff
@@ -18,6 +19,16 @@ fun XapiStatementEndpoint.storeProgressStatement(account: UmAccount, entry: Cont
             this.id = if (progress == 100) "https://w3id.org/xapi/adl/verbs/satisfied" else "http://adlnet.gov/expapi/verbs/progressed"
             this.display = mapOf("en-US" to if (progress == 100) "satisfied" else "progressed")
         }
+        this.context = XContext().apply {
```

Comment:
Why is there a context activity? Isn't this just progressed, where the object is the content piece (e.g. epub, video, etc)? I see why this needs the contextRegistration, but I don't see the need for contextActivities on this statement.